### ISSUE TITLE
[BUG] major accessibility issues

### ISSUE THREAD

**Issue by PlasticMemmories** [gkasdorf/memmy#120]
**Describe the bug**
There are a couple of accessibility issues in the app, including unlabeled elements and problems with reading comments with a screen reader.
**To Reproduce**
1. Go to 'settings'
2. Click on 'accessibility, then voiceover'
3. turn voiceover on. Voiceover changes the way you navigate your phone, so if using a newer device, swipe up until you here a click, then life your finger to go to your home screen. If you want your recent apps, swipe up until you here the second click
4. Open the app and navigate around it by swiping left/right

**Expected behavior**
the elements for subscriptions, sorting, and the element to the right of that, (i don't know how to describe what it does), should have labels that describe what they do, for example, "subscriptions" or "sort by" to allow screen reader users do navigate the app efficiently. There are also some unlabeled buttons when you open a post containing an image, and some unlabeled elements right next to every comment.
**Screenshots**

**Smartphone (please complete the following information):**
 - Device: iPhone SE, third addition
 - OS: [e.g. iOS17]
 - Version [0.1]

**Logs:**
If you wish to submit any debug logs, you may do so by doing the following:
 - Navigate to Settings
 - Tap version number 5 times
 - Press "Send Logs"

This will open an email in your mail app with the logs in the body of the email. 

If you want, you can post them here. However, although there should not be any PII inside of the logs, it may be best to just send them privately. If you do want to post them here, please either find and provide the applicable parts of the logs, or check them for any PII.

**Additional context**
this problem hinders usage of the app for blind/visually impaired users, making it hard for them to navigate lemmy efficiently. This app is in beta, so I wanted to attempt to get a fix for the accessibility issues before the app progresses. Other than that, the app seems amazing.

**Comment by gkasdorf** [comment]
I was talking to someone about accessibility stuff today. I definitely want to work this out, so I'll put this issue on the board. Thanks for the help.

**Comment by PlasticMemmories** [comment]
@gkasdorf I am so glad you're interested in fixing this problem, respect to you. I don't know where to add this in the issue but there's also a bug with comments where they're read out as one large block of text and not broken up into lines like how posts are, just thought I'd bring that to you're attention as well. Thanks once again!

**Comment by gkasdorf** [comment]
@PlasticMemmories this might get fixed once I finish comment optimizations. I'm working on something that will break down the comment rendering into a single comment regardless of whether they are chained or not. I suspect right now that might be part of the issue in that a whole chain is "one block" when each should be their own separate block.

**Comment by PlasticMemmories** [comment]
@ gkasdorf, take your time, thanks again.

**Comment by PlasticMemmories** [comment]
@gkasdorf sorry to keep commenting but I had an idea. Would you like me to open an issue that can act as a road map for all the accessibility issues? I can write up something that'd be edited whenever an accessibility issue is fixed. This will also let me organize the issues by heading, so you have a section for issues with posts, another one for comments, another one for miscellaneous  etc. I think it'd be more organized than simply filing a bug report, but I won't do it if it is redundant or not useful to you.

**Comment by sgriff96** [comment]
> @gkasdorf sorry to keep commenting but I had an idea. Would you like me to open an issue that can act as a road map for all the accessibility issues? I can write up something that'd be edited whenever an accessibility issue is fixed. This will also let me organize the issues by heading, so you have a section for issues with posts, another one for comments, another one for miscellaneous etc. I think it'd be more organized than simply filing a bug report, but I won't do it if it is redundant or not useful to you.

That may be worth it, especially if you are familiar with developing with accessibility in mind. Would be a big help for us.

**Comment by PlasticMemmories** [comment]
@sgriff96 Ok, I'll work on that today and open a new issue when I have everything written up.

**Comment by eoddc5** [comment]
Is this linked to a pr?
